### ISSUE TITLE
Deprecation warning with Ember 2.12.1

### ISSUE THREAD

**Issue by bentrility** [toranb/ember-cli-thymeleaf-csrf#5]
DEPRECATION: Addon files were detected in `blahblahblah/node_modules/ember-cli-thymeleaf-csrf/addon`, but no JavaScript preprocessors were found for `ember-cli-thymeleaf-csrf`. Please make sure to add a preprocessor (most likely `ember-cli-babel`) to in `dependencies` (NOT `devDependencies`) in `ember-cli-thymeleaf-csrf`'s `package.json`.

**Comment by toranb** [comment]
@bentrility ah good catch! seems we are running an ancient version of ember-cli here so I'll upgrade it and this should solve the issue found (sorry for the delay here)!

**Comment by toranb** [comment]
@bentrility I just pushed up a completely new addon using ember-cli v2.13.0-beta.3 and it seems to build as-is. Can you install this from the master branch to confirm it's deprecation free/working without issue (for both dev/ and prod builds)? If we are g2g I can publish a v0.3.0 tomorrow :)

```
npm install ssh://git@github.com:toranb/ember-cli-thymeleaf-csrf.git#master
```

**Comment by benkiefer** [comment]
Fixed, thank you. The stripe addon may need the same thing.

**Comment by toranb** [comment]
@benkiefer @bentrility good deal - just published v0.3.0 of this addon. The `ember-cli-patch-stripejs` addon is next on my list  :)

**Comment by benkiefer** [comment]
Thanks @toranb. Much appreciated. @bentrility is my work account :)
